src/main/java/io/confluent/connect/jdbc/source/TimestampIncrementingTableQuerier.java

```diff
@@ -171,9 +172,15 @@ public SourceRecord extractRecord() throws SQLException {
     final Map<String, String> partition;
     switch (mode) {
       case TABLE:
-        String name = tableId.tableName(); // backward compatible
-        partition = Collections.singletonMap(JdbcSourceConnectorConstants.TABLE_NAME_KEY, name);
-        topic = topicPrefix + name;
+        String name = tableId.tableName();
+        topic = topicPrefix + name;// backward compatible
+        String fqn = ExpressionBuilder.create().append(tableId, false).toString();
+        partition = new HashMap<>();
+        partition.put(JdbcSourceConnectorConstants.TABLE_NAME_KEY, fqn);
+        partition.put(
+            JdbcSourceConnectorConstants.OFFSET_PROTOCOL_VERSION_KEY,
+            JdbcSourceConnectorConstants.PROTOCOL_VERSION_ONE
+        );
```

Comment:
Didn't we talk about putting this logic into a helper method? Maybe an `OffsetProtocols` class with static methods:

    Map<String, String> sourcePartitionForProtocolV1(TableId table) { ... }
    Map<String, String> sourcePartitionForProtocolV0(TableId table) { ... }